LICENSE

```diff
@@ -0,0 +1,266 @@
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2001-2008 The Apache Software Foundation
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+=========================================================================
+
+This product bundles "Noto Sans" fonts, which are available under the
+"Apache License Version 2.0".
+For details, see framework/resources/fonts/NotoSans/.
+
+=========================================================================
+
+This product bundles the "Quicksand" fonts, which is available
+under the "SIL OFL License 1.1" license.
+For details, see themes/rainbowstone/webapp/rainbowstone/fonts/quicksand
+
+=========================================================================
+
+This product bundles "timezones" files, which are available in the "Public Domain".
+For details, see themes/common/webapp/common/js/plugins/date/timezones.
+
+=========================================================================
+
+This product bundles "Date Format 1.2.3", which is available
+under the "MIT" license. 
+For details, see themes/common/webapp/common/js/plugins/date/date.format-1.2.3*
+
```

Comment:
This section should be removed, because it references themes (which is not part of plugins).